test/contrib/test_tfp.py

```diff
@@ -53,6 +55,7 @@ def test_independent():
 
 
 @pytest.mark.filterwarnings("ignore:can't resolve package")
+@pytest.mark.filterwarnings("ignore:Importing distributions")
```

Comment:
This test can be removed I think

Reply:
Updated the test to check the pattern you describe.